pystats_docs.md

```diff
@@ -0,0 +1,81 @@
+# pystats documentation
+
+This describes the meaning of the various sections and fields in the pystats output.
+
+## Execution counts
+
+This is the counts of how many times each Tier 1 instruction is executed.
+
+The "miss ratio" column shows the percentage of times the instruction executed that it deoptimized.
```

Comment:
I’d like to know whether, upon a deopt, the “base” instruction is also counted. @markshannon ?

Reply:
The base instruction isn't counted.

Reply:
On a "deopt" we jump into the base instruction *after* stats have been recorded

Reply:
@mdboom Can you clarify this in the doc?